tests/tdastro/sources/test_snia.py

```diff
@@ -174,22 +154,22 @@ def test_snia_end2end(
         saltpars = {"x0": p["x0"], "x1": p["x1"], "c": p["c"], "z": p["redshift"], "t0": p["t0"]}
         model = sncosmo.Model(sncosmo_modelname)
```

Comment:
I'd like to find an alternate way to test the code. `sncosmo.Model` will download the model files each time the test is run in a new environment (including each run of the CI tests). This is already causing intermittent failures on github. We should isolate the tests from external downloads. We could do this by pre-downloading and including the model files or by comparing with known data.